src/map/options.ts

```diff
@@ -224,47 +224,39 @@ export class MapOptions extends OptionsGroup {
 
         const userFactor = logSlider(this.size.factor.value);
 
-        let values;
-        if (this.size.mode.value !== 'constant') {
```

Comment:
The whole if/else was no longer needed since the switch below covers the same case.